docs/cluster-management/configure-api.md

```diff
@@ -494,12 +494,19 @@ scms:
             email: dev-null@screwdriver.cd # [Optional] Email for code checkout
             commentUserToken: A_BOT_GITHUB_PERSONAL_ACCESS_TOKEN # [Optional] Token for writing PR comments in Github, needs "public_repo" scope
             privateRepo: false # [Optional] Set to true to support private repo; will need read and write access to public and private repos (https://developer.github.com/v3/oauth/#scopes)
+            autoDeployKeyGeneration: false # Set to true to allow automatic generation of private and public deploy keys and add them to the build pipeline and github respectively
```

Comment:
can you add this here too: https://github.com/screwdriver-cd/screwdriver/blob/master/config/default.yaml#L209
```suggestion
            autoDeployKeyGeneration: false # Set to true to allow automatic generation of private and public deploy keys and add them to the build pipeline and Github, respectively
```